### PR TITLE
Updated README.md to fit info as of 23/03/2021 (DD/MM/YYYY)

### PR DESCRIPTION
**Pull request by Cryptizism**
Updated:
`𝗔𝗯𝘆𝘀𝘀 𝗢𝘃𝗲𝗿𝗹𝗮𝘆 𝘃𝟭.𝟬.𝟬` ✧ to `𝗔𝗯𝘆𝘀𝘀 𝗢𝘃𝗲𝗿𝗹𝗮𝘆 𝘃𝟭.𝟬.𝟭 ✧`
- [x] Added new features to feature list
- [x] Added Upcoming Features
- [x] Added Known bugs and status 
- [x] Added a "Discord" header above info about the discord

Only made updates to README.md file.
Edits are allowed!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
-# 𝗔𝗯𝘆𝘀𝘀 𝗢𝘃𝗲𝗿𝗹𝗮𝘆 𝘃𝟭.𝟬.𝟬 ✧
+# 𝗔𝗯𝘆𝘀𝘀 𝗢𝘃𝗲𝗿𝗹𝗮𝘆 𝘃𝟭.𝟬.𝟭 ✧
```
Comment (Cryptizism) [inline README.md:1]: Updated version on title

File: README.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 
 ## Recommended Usage
-Open the overlay and put it on either the top left or top right of your screen. Select your client. Adjust any settings you want. Make sure to run `/who` in lobby when not all players in the lobby are registered in the overlay. Leave the rest up to the overlay :)
+Open the overlay and put it on either the top left or top right of your screen. Select your client. Adjust any settings you want. Make sure to run `/who` in lobby when not all players in the lobby are registered in the overlay. Leave the rest up to the overlay :) (Minecraft has to be in windowed as full screen is not supported yet)
```
Comment (Cryptizism) [inline README.md:4]: Adding the fact that full screen is not supported as of yet in usage

File: README.md [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
 
 ## Features
 - Shows stats of players in your lobby in seconds
+- Shows party stats
+- Shows stats of anyone that calls your name in lobby with an option to turn it on or off
```
Comment (Cryptizism) [inline README.md:9]: Added 2 new features

File: README.md [hunk 4]
```diff
@@ -21,6 +23,25 @@ Open the overlay and put it on either the top left or top right of your screen.
...
+- [ ] Linux support
+- [ ] Fullscreen (Will take awhile)
+- [ ] Manually add api key
+- [ ] Search for player stats
+- [ ] Improve sniper detection using login dates and other stats
+- [ ] Show guilds
+- [ ] Remember overlay position and size
+- [ ] Custom file path
+- [ ] Hover over names to see all stats
+- [ ] Game start timer
```
Comment (Cryptizism) [inline README.md:36]: Added upcoming features to make the overlay look more appealing and to show there is development

File: README.md [hunk 5]
```diff
@@ -21,6 +23,25 @@ Open the overlay and put it on either the top left or top right of your screen.
...
+
+## Known bugs + status
+
+- [ ] Scroll fix
+- [ ] Players showing as nicked when unable to contact api
+
```
Comment (Cryptizism) [inline README.md:42]: Added a known bugs and status to keep track of bugs easier

File: README.md [hunk 6]
```diff
@@ -21,6 +23,25 @@ Open the overlay and put it on either the top left or top right of your screen.
...
+## Discord
+
 The Abyss Overlay is made by Chit (OhChit). Join the [Discord](https://discord.gg/eDnHTT3aBS) and look in the overlay category to receive overlay updates, announcements, report bugs, and make suggestions. Any help is appreciated :D
```
Comment (Cryptizism) [inline README.md:45]: Added a header for discord to make it look better and more readable

**Review by Chit132 — APPROVED** [review]

**Pull request closed without merging**